smother/control.py

```diff
@@ -41,7 +46,7 @@ def start(self):
 
     def save_context(self, label):
         self.data[label] = {
-            key: sorted(map(int, val.keys()))
+            self._normalize_path(key): sorted(map(int, val.keys()))
```

Comment:
>One other thought: if the path alias logic were applied at read time instead of write time, I think we could eliminate the `--relative-paths` option -- report data would always be an absolute path, and the .coveragerc file could specify how to normalize those paths (potentially to relative paths if desired).

I have to add the `--relative-paths` option at some point no matter what. The way coverage's `PathAliases` class works, it always converts to abosolute paths, regardless of what the aliases are given. So if want to be able to produce a smother file, either by combining several or just regularly, that has relative paths, I have to call the function. It doesn't make sense to do it at read time, because it would be very expensive to the lookup.